contrib/coreos/user-data.example

```diff
@@ -133,3 +134,14 @@ write_files:
       TOOLBOX_DOCKER_IMAGE=ubuntu-debootstrap
       TOOLBOX_DOCKER_TAG=14.04
       TOOLBOX_USER=root
+  - path: /etc/environment_proxy
```

Comment:
Hmm, I'm wondering if it'd be better to throw settings like this into a deis-specific directory like `/etc/deis/environment` and call it the official place to store host-level envvars.